pep-0695.rst

```diff
@@ -809,24 +813,78 @@ It also adds an AST node type that represents a type parameter.
         | ParamSpec(identifier name)
         | TypeVarTuple(identifier name)
 
+Bounds and constraints are represented identically in the AST. In the implementation,
+any expression that is syntactically a tuple is treated as a constraint, and any other
```

Comment:
"syntactically a tuple" is a little opaque to me -- if you're referring to how this will be represented in the AST, I'd prefer it if you just said that :)